mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -1588,6 +1623,37 @@ struct GridwiseAttentionAccelRewritePattern
          elemTypeOut});
 
     zeroAccBuffer(rewriter, loc, attentionOutAccBuffer);
+#ifdef ROCK_DEBUG_ATTENTION_REMOVE_SOFTMAX
+    zeroAccBuffer(rewriter, loc, accRegBufferGemm1);
```

Comment:
I thought about it... I dont want this to be a functional thing. My thoughts were it needed to be very global switch that changes the behaviour in two places in the code base (here and rocmlir-gen) purely for debugging -- which I found useful : for e.g. that is how I figured blockwise_reductions are doing something wrong.

Op attribute would kinda make it a functional thing and I would need to more plumbing in rocmlir-gen reference generation. I kind of have the philosophy debugging capabilities should be simple as possible; so that one does not need to debug the debugging utilities.

What do you think ? or more precisely why do you think op attribute would be better ?

Reply:
I see your point about this being very much a debug switch, and thus something that's off in a source code edit ... though I might then take a page out of LLVM's book and make it a `ReallyHidden` CLI option instead.

I was thinking in the context of having GEMM-*-GEMM support in the future